lib/server_sent_event_stage.ex

```diff
@@ -224,43 +224,46 @@ defmodule ServerSentEventStage do
 
     uri = URI.parse(url)
 
-    scheme =
-      case uri.scheme do
-        "https" -> :https
-        "http" -> :http
-      end
+    response =
+      HTTP.connect(scheme_atom(uri.scheme), uri.host, uri.port, transport_opts: [timeout: 60_000])
```

Comment:
```suggestion
    response =
      uri.scheme |> scheme_atom() |> HTTP.connect(uri.host, uri.port, transport_opts: [timeout: 60_000])
```

Would it be worth using a small pipe chain?

Reply:
It could feed into `handle_connect_response` below even.